pkg/storage/cache/cache.go

```diff
@@ -71,11 +76,18 @@ func (cache *Cache) saveToDisk(key string, val interface{}) {
 }
 
 func (cache *Cache) Flush() {
+	// evict all the items in cache
 	cache.lfu.Evict(cache.lfu.Len())
+
 	close(cache.lfu.EvictionChannel)
+	// wait until cache flushing is finished
 	<-cache.cleanupDone
 }
 
+func (cache *Cache) Evit(percent float64) {
```

Comment:
Is this a typo? I think you meant `Evict`

Reply:
right~ sorry